.drone.yml

```diff
@@ -210,10 +243,47 @@ trigger:
   branch:
     - "staging"
     - "staging-next"
+    - "seth/fake-staging"
```

Comment:
Will remove 2x seth/fake-staging refs before merge, useful to have until this is final.